### DISCUSSION TITLE
Using EKS roles

### DISCUSSION THREAD

**Discussion by xorl** [kubero-dev/kubero#287] [Q&A]
Hello!

I am trying to figure out how I can modify the serviceaccount to use a role dynamically so that my EKS pods have permissions to specific resources. I have it handled on the AWS side but I am trying to get the serviceaccount to use the proper annotation for the pods to deploy under. Any help would be appreciated.

**Comment by mms-gianni** [comment]
The serviceaccount is created and handled by the operator for every app separately. 

They are basically a Helm chart: 
https://github.com/kubero-dev/kubero-operator/blob/main/helm-charts/kuberoapp/templates/serviceaccount.yaml 

Same for pod annotations: 
https://github.com/kubero-dev/kubero-operator/blob/main/helm-charts/kuberoapp/templates/deployment-web.yaml#L19 

They are not documented well yet, since I considered them a special case.
